packages/cacti-ledger-browser/package.json

```diff
@@ -74,7 +74,7 @@
     "@types/react-dom": "18.2.17",
     "@vitejs/plugin-react": "4.2.1",
     "typescript": "5.5.2",
-    "vite": "5.4.9"
+    "vite": "5.3.6"
```

Comment:
This is getting downgraded for some reason

Reply:
Fixed manually

Reply:
@outSH Oops, sorry I missed that one! Thank you for the fix! I'll take mental note for next time that it's not just axios that the robot tries to downgrade but also vite.